### PR TITLE
Add Job cpus getter

### PR DESCRIPTION
**Pull request by orlandohohmeier**
/cc @jfurrow

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

File: src/js/structs/Job.js [hunk 1]
```diff
@@ -10,6 +10,12 @@ module.exports = class Job extends Item {
     return this.get('run').cmd;
   }
 
+  getCpus() {
+    const {cpus = 0.01} =  this.get('run') || {};
```
Comment (aldipower) [inline src/js/structs/Job.js]: Hehe. ;)

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

File: src/js/structs/Job.js (hunk 1 above)
Comment (MatApple) [inline src/js/structs/Job.js]: We've got an extra space after the `=` :)

**Comment by MatApple** [comment]
@orlandohohmeier just one comment about the extra space and we can merge! :)

File: src/js/structs/Job.js (hunk 1 above)
Comment (rcorral) [inline src/js/structs/Job.js]: Could we add a comment for the `0.01` why it's that and not just `0`?

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

File: src/js/structs/Job.js [hunk 2]
```diff
@@ -12,6 +12,13 @@ module.exports = class Job extends Item {
     return cmd;
   }
 
+  getCpus() {
+    // Default to the minimum number of cpus per Mesos offer
```
Comment (pierlo-upitup) [inline src/js/structs/Job.js]: I wonder if we should move this (and all other) default Mesos value to a constant file `MesosDefaults`. Out of scope, I guess.

Comment (orlandohohmeier) [inline src/js/structs/Job.js]: Yeah, thats a really good idea! But as you guessed not in the scope of this PR. I'll create a tech debt issue.

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Pull request merged**